app/server/services/sms.js

```diff
@@ -0,0 +1,26 @@
+// Download the helper library from https://www.twilio.com/docs/node/install
+// Your Account Sid and Auth Token from twilio.com/console
+const accountSid = process.env.TWILIO_ACCOUNT_SID;
+const authToken = process.env.TWILIO_AUTH_TOKEN;
```

Comment:
We'll need these changes to the `.env` shared with devs and updated on production

Reply:
> Is there a limit of phone numbers that we can send to at once? We got that issue raised while creating the mass email system.
> 
> Otherwise the implementation looks good. Will test again before merging after this PR is approved.

From what I understand here https://support.twilio.com/hc/en-us/articles/223183648-Sending-and-Receiving-Limitations-on-Calls-and-SMS-Messages, it doesn't seem like there is a limit. It just may take some time to send it to everyone since it's 1 message per API call and the calls get queued on their server. To speed things up, we could purchase more than one phone number, but I think we'll be few enough people that one number will be fine.